src/server/internalServer.cpp

```diff
@@ -353,6 +353,7 @@ SuggestionsList_t getSuggestions(const zim::Archive* const archive,
     // TODO: This case should be handled by libzim
     std::vector<std::string> variants = getTitleVariants(queryString);
     int currCount = 0;
+    int suggestionCount = end - start;
```

Comment:
Getting results of a certain range in non-Xapian based search is not supported as of now, so we will simply consider the count of suggestions here. 
This will anyway have to be implemented on libzim side after openzim/libzim#574